stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -183,22 +186,43 @@ class Worksheet(_WorkbookChild):
     @property
     def columns(self) -> Generator[tuple[Cell, ...], None, None]: ...
     def set_printer_settings(
-        self, paper_size: int | None, orientation: None | Literal["default", "portrait", "landscape"]
+        self, paper_size: int | None, orientation: Literal["default", "portrait", "landscape"] | None
     ) -> None: ...
     def add_data_validation(self, data_validation: DataValidation) -> None: ...
-    def add_chart(self, chart, anchor: Incomplete | None = None) -> None: ...
-    def add_image(self, img, anchor: Incomplete | None = None) -> None: ...
+    def add_chart(self, chart: ChartBase, anchor: str | None = None) -> None: ...
+    def add_image(self, img: Image, anchor: str | None = None) -> None: ...
     def add_table(self, table: Table) -> None: ...
     @property
     def tables(self) -> TableList: ...
     def add_pivot(self, pivot) -> None: ...
+    # Same overload as CellRange.__init__
+    @overload
     def merge_cells(
         self,
-        range_string: str | None = None,
-        start_row: int | None = None,
-        start_column: int | None = None,
-        end_row: int | None = None,
-        end_column: int | None = None,
+        range_string: str,
+        start_row: Unused = None,
+        start_column: Unused = None,
+        end_row: Unused = None,
+        end_column: Unused = None,
```

Comment:
I wonder whether it would be safer to use `: None = None` here. It seems providing `range_string` and the rows and cols - while working at runtime - could introduce subtle bugs and is probably not what anyone wants.

But we should at least use `ConvertibleToInt | None` instead of `Unused`, because providing anything else (while working) is surely a bug.

Reply:
Ok. I can see how passing both (which this allows) may lead to unexpected bug by not knowing which param is actually used. I'll move the concept that these params go unused in that overload as a comment instead of the parameter type.